tests/console/gpg.pm

```diff
@@ -57,6 +57,14 @@ EOF
         );
         assert_script_run("cat $egg_file");
 
+        # Kill gpg-agent service when executing gpg2 command in case gpg-agent
+        # does NOT see your environment variable: LIBGCRYPT_FORCE_FIPS_MODE=1
+        # when gpg version > 2.1
+        # Refer to bug #1198135
+        if (get_var('FIPS_ENV_MODE')) {
+            assert_script_run('gpgconf --kill gpg-agent');
```

Comment:
I think this should be also fine.

`assert_script_run('gpgconf --kill gpg-agent') if (get_var('FIPS_ENV_MODE')); `